admin/modules/tools/maillogs.php

```diff
@@ -300,7 +300,12 @@
 		$log['subject'] = htmlspecialchars_uni($log['subject']);
 		$log['dateline'] = my_date('relative', $log['dateline']);
 
-		if($log['type'] == 2)
+		if($log['type'] == 1)
```

Comment:
More magic numbers 😢 We seem to really like these in MyBB. A nice quality of life improvement for anyone interested would be a PR to replace these at some point in the future...